include/JSystem/JGadget/std-list.hpp

```diff
@@ -107,10 +123,15 @@ template <class T, class A = TAllocator<T> > class TList {
 		return start;
 	}
 
+	bool Confirm()
+	{
+		// TODO
+	}
```

Comment:
is this needed for ordering? if not, move it to private part of the class please